ledger/src/shred/wire.rs

```diff
@@ -434,6 +442,36 @@ mod tests {
         Signature::from(signature)
     }
 
+    fn write_shred<R: Rng>(
+        rng: &mut R,
+        shred: impl AsRef<[u8]>,
+        nonce: Option<Nonce>,
+        packet: &mut Packet,
+    ) {
+        let buffer = packet.buffer_mut();
+        let capacity = buffer.len();
+        let mut cursor = Cursor::new(buffer);
+        cursor.write_all(shred.as_ref()).unwrap();
+        // Write some random many bytes trailing shred payload.
```

Comment:
nit: I think no need for `many` here ?

Reply:
I meant here the "number" of bytes written are also random.
anyways, flaky tests have made CI very slow, and I would rather not to kick off CI again for this.

Reply:
Fair enough, the code is simple enough so a reader can still easily glean what is happening